frontend/src/tests/utils/renderWithRouter.js

```diff
@@ -0,0 +1,28 @@
+import configureStore from 'redux-mock-store'
+import { createMemoryHistory } from 'history'
+import { render } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import { Route, Router, Switch } from 'react-router-dom'
+import React from 'react'
+
+const renderWithRouter = (component, historyPathArray = ['/']) => {
```

Comment:
Since its also provides a store, consider renaming to `renderWithStoreAndRouter`

Reply:
:ok_hand: Renamed